modules/react/breadcrumbs/lib/BreadcrumbsItem.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {
+  composeHooks,
+  createSubcomponent,
+  createElemPropsHook,
+} from '@workday/canvas-kit-react/common';
+import {
+  useListItemRegister,
+  useOverflowListItemMeasure,
+} from '@workday/canvas-kit-react/collection';
+import {Flex} from '@workday/canvas-kit-react/layout';
+import {SystemIcon} from '@workday/canvas-kit-react/icon';
+import {chevronLeftSmallIcon, chevronRightSmallIcon} from '@workday/canvas-system-icons-web';
+import {colors} from '@workday/canvas-kit-react/tokens';
+import {useRTL} from './hooks/useRTL';
+import {useBreadcrumbsModel} from './hooks/useBreadcrumbsModel';
+import {BreadcrumbsLink} from './BreadcrumbsLink';
+
+export interface BreadcrumbsItemProps {
+  /**
+   * The contents of the action item. This will be the accessible name of the action for screen readers.
+   *
+   * ```tsx
+   * <Breadcrumbs.Item>First Action</Breadcrumbs.Item>
+   * ```
+   */
+  children: React.ReactNode;
+  /**
+   * The identifier of the action. This identifier will be used for correct overflow behavior.
+   * If this property is not provided, it will default to a string representation
+   * of the the zero-based index of the Item when it was initialized.
+   */
+  'data-id'?: string;
+}
+
+export const useBreadcrumbsItem = composeHooks(
+  useOverflowListItemMeasure,
+  useListItemRegister,
+  createElemPropsHook(useBreadcrumbsModel)(
+    (
+      {state},
+      _?: React.Ref<HTMLElement>,
+      elemProps: {
+        'data-id'?: string;
+        item?: {id: string};
+      } = {}
+    ) => {
+      const [localId] = React.useState(elemProps['data-id'] || elemProps.item?.id || '');
+
+      return {
+        inert: state.nonInteractiveIds.includes(localId) ? '' : undefined,
+        disabled: undefined,
+      };
+    }
+  )
+);
+
+export const BreadcrumbsItem = createSubcomponent('li')({
+  displayName: 'Breadcrumbs.Item',
+  modelHook: useBreadcrumbsModel,
+  elemPropsHook: useBreadcrumbsItem,
+  subComponents: {
+    Link: BreadcrumbsLink,
+  },
+})<BreadcrumbsItemProps>(({children, ...elemProps}, Element) => {
+  const {shouldUseRTL} = useRTL();
+  const icon = shouldUseRTL ? chevronLeftSmallIcon : chevronRightSmallIcon;
```

Comment:
I think we have a hook for this `useIsRTL` https://github.com/Workday/canvas-kit/blob/master/modules/react/common/lib/theming/useIsRTL.ts